assets/css/layout/_layout.scss

```diff
@@ -95,7 +95,7 @@ figcaption {
 
 	.content-wrapper {
 		display: grid;
-		grid-template-columns: 1fr minmax(75ch, 130ch) 1fr;
+		grid-template-columns: 1fr minmax(75ch, 130ch) 23ch;
```

Comment:
What's the rationale for this change? It makes the right column a fixed width, which leads to this kind of weirdness on large screens:

<img width="2503" alt="Screen Shot 2020-04-22 at 8 43 50 AM" src="https://user-images.githubusercontent.com/605361/79996212-6a703500-8475-11ea-9c87-127384a77225.png">

Reply:
@greatislander the left side of the background is uneven otherwise. But I see it is causing other problems. I spent a lot of time on this yesterday. Can we do a one on one? I need help figuring this out.